ci/docs/build.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Copyright (c) 2021, NVIDIA CORPORATION.
+######################################
+# Dask-CUDA Docs build script for CI #
+######################################
+
+if [ -z "$PROJECT_WORKSPACE" ]; then
+    echo ">>>> ERROR: Could not detect PROJECT_WORKSPACE in environment"
+    echo ">>>> WARNING: This script contains git commands meant for automated building, do not run locally"
+    exit 1
+fi
+
+export DOCS_WORKSPACE=$WORKSPACE/docs
+export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
+export HOME=$WORKSPACE
+export PROJECT_WORKSPACE=/rapids/dask-cuda
+export NIGHTLY_VERSION=$(echo $BRANCH_VERSION | awk -F. '{print $2}')
+export PROJECTS=(dask-cuda)
+
+gpuci_logger "Check environment..."
+env
+
+gpuci_logger "Check GPU usage..."
+nvidia-smi
+
+gpuci_logger "Activate conda env..."
+. /opt/conda/etc/profile.d/conda.sh
+conda activate rapids
```

Comment:
Are there any packages we would need to add to this environment to get docs builds working?

https://github.com/rapidsai/dask-cuda/blob/d39fff9cd6f766dc03cb9f6858b19caf7d3cc438/conda/environments/builddocs_py37.yml#L5-L9

Reply:
The docs are built in the `devel` images and automatically download some basic stuff such as rapids-doc-env. I don't think `sphinx-click` is included in our `devel` environment so this may need to be added manually.

Reply:
Would this need to be added from the script, or could it be added to the `devel` environment permanently?